pkg/drivers/kic/oci/network.go

```diff
@@ -168,42 +149,152 @@ func dockerContainerIP(name string) (string, string, error) {
 	return ips[0], ips[1], nil
 }
 
-// CreateNetwork creates a network
-func CreateNetwork(name, ipRange, gateway string) error {
+// CreateNetwork creates a network returns subnet and error
+func CreateNetwork(name string) (*net.IPNet, error) {
 	// check if the network already exists
-	if networkExists(name) {
-		return nil
+	subnet, _, err := dockerNetworkInspect(name)
+	if err == nil {
+		return subnet, nil
 	}
 
-	subnet := fmt.Sprintf("--subnet=%s", ipRange)
-	_, err := runCmd(exec.Command(Docker, "network", "create", "--driver=bridge", subnet, "--gateway", gateway, name))
+	// simple way to create networks, subnet is taken, try one bigger
+	attempt := 0
+	_, subnet, err = net.ParseCIDR(defaultSubnet)
 	if err != nil {
-		return errors.Wrapf(err, "error creating network")
+		return nil, errors.Wrapf(err, "parse default subnet %s", defaultSubnet)
+	}
+
+	err = attemptCreateNework(subnet, name)
+	if err != nil {
+		if err == ErrNetworkSubnetTaken {
```

Comment:
exit early if err != ErrNetworkSubnetTaken, so that the longer case has the minimum indentation.